src/main/webapp/WEB-INF/jsp/merchant.jsp

```diff
@@ -40,23 +49,24 @@
 							class="glyphicon glyphicon-chevron-right"></span> Debit</a></li>
 					<li id="transfer"><a href="transfer"><span
 							class="glyphicon glyphicon-chevron-right"></span> Transfer</a></li>
-					<li id="addAccount"><a href="regular"><span
+					<li id="addAccount"><a href="#"><span
 							class="glyphicon glyphicon-chevron-right"></span> Add Account</a></li>
-					<li id="transactions"><a href="searchTransaction"><span
+					<li id="transactions"><a href="externalsearchtrans"><span
 							class="glyphicon glyphicon-chevron-right"></span> Transactions</a></li>
-					<li id="notifications"><a href="searchTransaction"><span
+					<li id="bankStatement"><a href="bankStatement"><span
+							class="glyphicon glyphicon-chevron-right"></span> Bank Statement</a></li>
+					<li id="notifications"><a href="notifications"><span
 							class="glyphicon glyphicon-chevron-right"></span> Notifications</a></li>
 					<li id="createRequest"><a href="#"><span
 							class="glyphicon glyphicon-chevron-right"></span> Create Request</a></li>
-					<li id="logout"><a href="#"><span
```

Comment:
OK. I did not know that.
Please change the code for Merchant and Individual also then.

On Sun, Oct 18, 2015 at 7:13 PM, Nagarjuna Myla <notifications@github.com>
wrote:

> In src/main/webapp/WEB-INF/jsp/merchant.jsp
> <https://github.com/chasethenag420/sbs/pull/72#discussion_r42331997>:
>
> >  							class="glyphicon glyphicon-chevron-right"></span> Notifications</a></li>
> >  					<li id="createRequest"><a href="#"><span
> >  							class="glyphicon glyphicon-chevron-right"></span> Create Request</a></li>
> > -					<li id="logout"><a href="#"><span
>
> Why you remove this? I did this form should be submitted..This should be
> inside form else logout will not work
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/chasethenag420/sbs/pull/72/files#r42331997>.
>



-- 
Thanks and Regards,
*Ganesh R Zilpe*
*Graduate Student(Computer Science)*
*Arizona State University*
*ganesh.zilpe@asu.edu <ganesh.zilpe@asu.edu>  |  +1-480-289-0918*